### PR TITLE
Added Community Forum in README file, added some extra formatting also

### PR DESCRIPTION
**Pull request by devansh42**
Signed-off-by: Devansh Gupta <devanshguptamrt@gmail.com>

**Description**

* Added community forum within README file
* Added some formatting also

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,11 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
...
 
 <p>
-✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">weekly community meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
-✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">meeting recordings</a>.<br />
-✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
+✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">Weekly Community Meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
+✔️ <em><strong>Watch</strong></em> Community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Meeting Recordings</a>.<br />
+✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">Community Drive</a>.<br />
+✔️ <em><strong>Discuss</strong></em> in the <a href="https://discuss.layer5.io">Community Forum</a>.<br />
+
```
Comment (aminoxix) [inline README.md]: @devansh42 : Pl remove this space b/w. `Line 25` & `Line 26`.
Also, signed-off your commit by clicking <a href="https://github.com/layer5io/gowrk2/pull/12/checks?check_run_id=2965655174">here</a>, & simply run the command in your code editor shell.

**Review by aminoxix — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (devansh42) [reply README.md]: I have made the changes as you asked for.

Comment (aminoxix) [reply README.md]: @devansh42 : Also need to signed-off your commit, as said previously :slightly_smiling_face: 
Wanna learn more, go through <a href="https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md">this</a>.

Comment (devansh42) [reply README.md]: I'm done with signing off the commits. Pardon me for these noob activities.

Comment (aminoxix) [reply README.md]: @devansh42: I think, you didn't notice that `unverified` signature, Pl check whether you are using right means to commit.

@adithyaakrishna : Sir, pl take this to your end. :slightly_smiling_face:

Comment (devansh42) [reply README.md]: @amino19 I have added my gpg key, it's now verified.

**Review by adithyaakrishna — APPROVED** [review]

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]
@amino19 Isn't this supposed to be like this? 👀
image

A readme for reference, https://github.com/layer5io/layer5/blob/master/README.md#-community-and-contributing

**The one right now in this PR is like this,**

image

**Comment by adithyaakrishna** [comment]
IMO what I thought was this would be a uniform template or structure for the `Community & Contributing` Section across all Layer5 Repos

**Comment by aminoxix** [comment]
@adithyaakrishna : Thanks for raising this out! My bad, I missed! Thanks :slightly_smiling_face:

File: README.md [hunk 2]
```diff
@@ -19,9 +19,10 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
...
 
 <p>
-✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">weekly community meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
-✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">meeting recordings</a>.<br />
-✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
+✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">Weekly Community Meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
+✔️ <em><strong>Watch</strong></em> Community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Meeting Recordings</a>.<br />
+✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">Community Drive</a>.<br />
```
Comment (shuklaritvik06) [inline README.md]: I think you should write "Access the community drive by completing a community member form".

File: README.md [hunk 3]
```diff
@@ -19,9 +19,10 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
 <a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src=".github/readme/images/community.png" style="margin-right:8px;padding-top:5px;" width="140px" align="left" /></a>
 
 <p>
-✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">weekly community meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
-✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">meeting recordings</a>.<br />
-✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
+✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">Weekly Community Meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
```
Comment (amino19) [inline README.md:22]: @devansh42 : Need to take reference from <a href="https://github.com/layer5io/layer5/blob/master/README.md">Layer5 Readme.md</a>.

File: README.md (hunk 2 above)
Comment (amino19) [reply README.md:24]: @devansh42 : By extending @includestdlibrary suggestion, you have to take same reference as mentioned above with Layer5. :slightly_smiling_face:

**Review by amino19 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Good of you to stick with this @devansh42 and see it through.

**Comment by leecalcote** [comment]
@devansh42 how is this coming along?

File: README.md (hunk 3 above)
Comment (aminoxix) [inline README.md]:
```suggestion
✔️ <em><strong>Join</strong></em> any or all of the weekly meetings on <a href="https://calendar.google.com/calendar/b/1?cid=bGF5ZXI1LmlvX2VoMmFhOWRwZjFnNDBlbHZvYzc2MmpucGhzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20">community calendar</a>.<br />
```

File: README.md (hunk 2 above)
Comment (aminoxix) [reply README.md]:
```suggestion
✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">Community Drive</a> by completing a community <a href="https://layer5.io/newcomer">Member Form</a>.<br />
```

File: README.md [hunk 4]
```diff
@@ -19,9 +19,10 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
...
+✔️ <em><strong>Watch</strong></em> Community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Meeting Recordings</a>.<br />
+✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">Community Drive</a>.<br />
+✔️ <em><strong>Discuss</strong></em> in the <a href="https://discuss.layer5.io">Community Forum</a>.<br />
```
Comment (aminoxix) [inline README.md:25]: @devansh42 : Thanks for adding this! :slightly_smiling_face: 
Need some sort of changes!

**Review by aminoxix — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (aminoxix) [reply README.md]: @devansh42 : Here's the final chnages! :slightly_smiling_face:

File: README.md (hunk 4 above)
Comment (devansh42) [reply README.md:25]: @amino19 I'm a little bit confused here, please help me out in resolving these pull request issues, like what exactly we want or if there is an example implementation.

Comment (aminoxix) [reply README.md:25]: @devansh42 : I suggest you the changes above; for reference you can add points as similar to <a href="https://github.com/meshery/meshery/blob/master/README.md/"> this</a>!

Comment (adithyaakrishna) [reply README.md:25]: @devansh42 Could you ping me on slack?
We can get on a call tomorrow and I can help you resolve this :)

Comment (aminoxix) [reply README.md:25]: @devansh42 : Thanks, the current commit looks good! :)

Comment (devansh42) [reply README.md:25]: @amino19 Shall we merge this request?

**Review by shuklaritvik06 — APPROVED** [review]
CITY :)

**Review by adithyaakrishna — APPROVED** [review]

**Comment by adithyaakrishna** [comment]
LGTM 🎉

File: README.md (hunk 4 above)
Comment (aminoxix) [reply README.md:25]: @devansh42 : Clearly, @adithyaakrishna have a crown here!
Pl take this to you end :slightly_smiling_face: 

LGTM! :partying_face:

**Review by aminoxix — APPROVED** [review]

**Pull request merged**